keyboards/newgame40/keymaps/default/keymap.c

```diff
@@ -0,0 +1,213 @@
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+ #include QMK_KEYBOARD_H
+ #include "eeconfig.h"
+
+ extern keymap_config_t keymap_config;
+
+ // Each layer gets a name for readability, which is then used in the keymap matrix below.
+ // The underscores don't mean anything - you can have a layer called STUFF or any other name.
+ // Layer names don't all need to be of the same length, obviously, and you can also skip them
+ // entirely and just use numbers.
+ #define _QWERTY 0
+ #define _COLEMAK 1
+ #define _DVORAK 2
+ #define _LOWER 3
+ #define _RAISE 4
+ #define _EUCALYN 5
+ #define _ADJUST 16
+
+ enum custom_keycodes {
+   QWERTY = SAFE_RANGE,
+   COLEMAK,
+   DVORAK,
+   LOWER,
+   RAISE,
+   EUCALYN,
+   ADJUST,
+ };
+
+ // Fillers to make layering more clear
+ #define _______ KC_TRNS
+ #define XXXXXXX KC_NO
+ #define LOWER MO(_LOWER)
+ #define RAISE MO(_RAISE)
+
+ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+ /* Qwerty
+  * ,---------------------------------------------------------------------.
+  * |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  |
+  * |------+------+------+------+------+------|------+------+------+------|
+  * |   A  |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  |   -  |
+  * |------+------+------+------+------+------+-------------+------+------|
+  * |   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |   /  |
+  * |-------------+------+------+------+------+------+------+------+------'
+  * |Shift |  GUI |Lower | Ctrl |Space | Bksp |Enter |Raise | Left |Right |
+  * `---------------------------------------------------------------------'
+  */
+ [_QWERTY] = LAYOUT( \
+   KC_Q,     KC_W,     KC_E,      KC_R,      KC_T,      KC_Y,      KC_U,      KC_I,      KC_O,      KC_P,     \
+   KC_A,     KC_S,     KC_D,      KC_F,      KC_G,      KC_H,      KC_J,      KC_K,      KC_L,      KC_MINS,  \
+   KC_Z,     KC_X,     KC_C,      KC_V,      KC_B,      KC_N,      KC_M,      KC_COMM,   KC_DOT,    KC_SLSH,  \
+   KC_LSFT, KC_LGUI, LT(_LOWER,KC_LANG2),  KC_LCTL, KC_SPC,  KC_BSPC, KC_ENT, LT(_RAISE,KC_LANG1),  KC_LEFT, KC_RGHT\
+ ),
+
+ /* Colemak
+  * ,---------------------------------------------------------------------.
+  * |   Q  |   W  |   F  |   P  |   G  |   J  |   L  |   U  |   Y  |   -  |
+  * |------+------+------+------+------+------|------+------+------+------|
+  * |   A  |   R  |   S  |   T  |   D  |   H  |   N  |   E  |   I  |   O  |
+  * |------+------+------+------+------+------+-------------+------+------|
+  * |   Z  |   X  |   C  |   V  |   B  |   K  |   M  |   ,  |   .  |   /  |
+  * |-------------+------+------+------+------+------+------+------+------'
+  * |Shift |  GUI |Lower | Ctrl |Space | Bksp |Enter |Raise | Left |Right |
+  * `---------------------------------------------------------------------'
+  */
+ [_COLEMAK] = LAYOUT( \
```

Comment:
If you're using `LAYOUTS = ortho_4x10`, then you should use that here: 

```suggestion
 [_COLEMAK] = LAYOUT_ortho_4x10( \
```